app/models/participant_declaration.rb

```diff
@@ -54,4 +54,13 @@ def refresh_payability!
       reload
     end
   end
+
+  def voided
```

Comment:
Don't like this in here if we're allowing multiple voids on the same record. Are we just going to keep the first one of those even if we don't reject them?

Reply:
I don't think this PR wants to allow multiple voids per single record. I am pretty sure more than one void is not a valid business state in general.